spark-etl/src/main/scala/geotrellis/spark/etl/config/Credentials.scala

```diff
@@ -0,0 +1,24 @@
+package geotrellis.spark.etl.config
+
+case class Credentials(accumulo: List[Accumulo], cassandra: List[Cassandra], s3: List[S3], hadoop: List[Hadoop]) {
```

Comment:
The job of this class seems to be to parse out the backend specific parameters, this is something that should be handled by constructor or companion object of each particular back-end.